about/index.md

```diff
@@ -6,29 +6,30 @@ last_modified_at: February 22, 2021
 sidenav: about_us
 ---
 
-Search.gov is a by-government, for-government web search engine. Available at no cost to federal agencies and the District of Columbia, we currently support the search boxes on about 2,200 websites, across 30% of federal domains. We are a product of Smarter IT Solutions, a division of the U.S. General Services Administration’s Technology Transformation Services, 
+Search.gov is a search engine built by the government, for the government. Available to federal agencies and the District of Columbia, we currently support over 300 million search queries a year for about 2,200 websites, across one third of federal domains.
```

Comment:
Is the District of Columbia still accurate?

Reply:
Yes, the 'available to' is the critical point here. We are still available to them.